### PR TITLE
skaffold: 0.30.0 -> 0.34.1

### PR DESCRIPTION
**Pull request by vdemeester**
###### Motivation for this change

Bump to latest release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by vdemeester** [comment]
@GrahamcOfBorg build skaffold

File: pkgs/development/tools/skaffold/default.nix [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 buildGoPackage rec {
   name = "skaffold-${version}";
-  version = "0.30.0";
-  # rev is the 0.30.0 commit, mainly for skaffold version command output
+  version = "0.33.0";
+  # rev is the 0.33.0 commit, mainly for skaffold version command output
   rev = "fe31429012110e6fd70f97971288bd266ba95bed";
```
Comment (teto) [inline pkgs/development/tools/skaffold/default.nix]: is it normal that the rev wasn't changed even though the comment points at another version ?

File: pkgs/development/tools/skaffold/default.nix [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 buildGoPackage rec {
   name = "skaffold-${version}";
```
Comment (teto) [inline pkgs/development/tools/skaffold/default.nix]: you could change it to `pname =  "skaffold";`

**Comment by vdemeester** [comment]
@GrahamcOfBorg build skaffold

**Comment by worldofpeace** [comment]
obsoleted by https://github.com/NixOS/nixpkgs/pull/66691

**Pull request closed without merging**